### DISCUSSION TITLE
add support for seller/*/mywants

### DISCUSSION THREAD

**Discussion by magnaux** [Purfview/Discogs-Scout#11] [Ideas]
https://www.discogs.com/seller/*/mywants?ev=hxiiw

**Comment by Purfview** [comment]
This would be like redundant duplicate for mywantlist page.

**Reply by magnaux** [reply]
but for the items sold by specific seller !

**Comment by magnaux** [comment]
![εικόνα](https://user-images.githubusercontent.com/107857394/213647730-9cc21d76-0d2d-4109-af32-4f5636624bbb.png)
the little box with Want / Have could be replaced with check results 

would be nice to see if for example if its a request before buying ... (my wantlist is 6000+ so its too hard to remember )
ty and thanks for understanding
